Blockzilla/BrowserViewController.swift

```diff
@@ -624,7 +624,15 @@ class BrowserViewController: UIViewController {
     }
 
     fileprivate func presentImageActionSheet(title: String, link: String?, saveAction: @escaping () -> Void, copyAction: @escaping () -> Void) {
-        let alertController = UIAlertController(title: title.truncated(limit: 160, position: .middle), message: nil, preferredStyle: .actionSheet)
+
+        let truncateLimit = 160
+        let leader = "..."
+        let headCharactersCount = (truncateLimit - leader.count) / 2
+        let tailCharactersCount = Int(ceil(Double(truncateLimit - leader.count) / 2.0))
+
+        let truncatedTitle = title.count <= truncateLimit ? title : String("\(title.prefix(headCharactersCount))\(leader)\(title.suffix(tailCharactersCount))")
```

Comment:
This should be an if check prior to doing anything else. 
```
let title = title
if (title.count > truncateLimit) {
    let leader = "..."
    let headCharactersCount = (truncateLimit - leader.count) / 2
    let tailCharactersCount = Int(ceil(Double(truncateLimit - leader.count) / 2.0))
    title = String("\(title.prefix(headCharactersCount))\(leader)\(title.suffix(tailCharactersCount))")
}
```

Reply:
Was just thinking, actually, maybe leader itself should be a constant in UIConstants.strings ...meaning that both head and tail characterscounts could also be constants. @sblatz would having leader inside UIConstants.strings be a pain for localization (since it doesn't need to be localized)?

Reply:
In my opinion it wouldn't be painful because we already have some strings like:

`static let sumoTopicUsageData = "usage-data"
static let encodingNameUTF8 = "utf-8"`

One question, the characters counts, should be under the UIConstants.layout or UIConstants.strings?

Reply:
@leschlogl Awesome! Let's put character counts inside UIConstants.layout - thank you!